models/sipnet/R/write_restart.SIPNET.R

```diff
@@ -89,6 +89,12 @@ write_restart.SIPNET <- function(outdir, runid, start.time, stop.time, settings,
     names(analysis.save[[length(analysis.save)]]) <- c("litter_mass_content_of_water")
   }
   
+  if ("SoilMoist" %in% variables) {
+    analysis.save[[length(analysis.save) + 1]] <- new.state$SoilMoist
```

Comment:
I'm surprised that write_restart doesn't include a conversion of SoilMoist back to SoilMoistFrac

Reply:
There is "write.configs" function called in "write_restart" function to do the conversion of SoilMoist back to SoilMoistFrac.